miqa/core/tests/pyppeteer/test_import_export.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+
+
+async def get_current_num_scans(page):
+    return len(await page.xpath('//ul[contains(@class, "scans")]/li'))
+
+
+async def edit_import_export_paths(page):
+    # Edit import path
+    import_path_input = await page.waitForXPath('//input[contains(@name, "import-path")]')
+    await import_path_input.click({'clickCount': 3})  # highlight the text to overwrite it
+    await import_path_input.type('samples/demo_project.csv')
+    await page.waitFor(1_000)
+
+    # Edit export path
+    import_path_input = await page.waitForXPath('//input[contains(@name, "export-path")]')
+    await import_path_input.click({'clickCount': 3})  # highlight the text to overwrite it
+    await import_path_input.type('samples/demo_project_export.csv')
+    await page.waitFor(1_000)
+
+    # Click save
+    await (await page.waitForXPath('//button[span[.=" Save "]][@type="submit"]')).click()
+    await page.waitFor(1_000)
```

Comment:
If there's a save confirmation notification, it would be good to check for that here.